### PR TITLE
Cu 864dzkz4y Refactor utils & Planner Card component

### PR DESCRIPTION
**Pull request by iguoliveira**
##### Description of changes

- [x] Why (Why did you make this pull request): Create a new component to get started with planner index page and refactored code in navbar and sidebar.
- [x] Brief description of the changes: Created the project card component, but the logic for edit the project (name) isn't yet implemented. Refactor the code in navbar and sidebar. Fixed layout errors when using sidebar and navbar together. Created a custom hook 'useToggle'.
- [x] Possible breakouts (Possible places where things could break): Need to use all components in index to see if layout is working how its supposed to be.

### CODE REVIEW

File: .env.dev [hunk 1]
```diff
@@ -11,7 +11,7 @@ DATABASE_URL=file:./dev.db
   # ## S3
     SERVICE_S3_PORT=5001
     # VITE_SERVICE_S3_URL=http://localhost:5001
-    VITE_SERVICE_S3_URL=http://localhost:5001
+    VITE_SERVICE_S3_URL=https://5001-etseiner-einer-kr164asyyu4.ws-us87.gitpod.io
```
Comment (viktormarinho) [inline .env.dev]: arrumar

**Review by viktormarinho — APPROVED** [review]
🚀 ok! 👍  👍  👍

**Review by gustavobaltazar — APPROVED** [review]
👍  clean.

**Review by gustavobaltazar — APPROVED** [review]
👍 Bullet.

**Pull request merged**